serializer/serializer.c

```diff
@@ -1,5 +1,11 @@
 #include "serializer.h"
 
+void fill_tag(char* new_tag, char* old_tag, size_t tag_len) {
+  char nines[27] = "999999999999999999999999999";
+  int pad_len = 27 - (int)tag_len;
```

Comment:
Change `27` to `sizeof(nines)`.

Reply:
I'll modify to `NUM_TRYTES_TAG`, that will be more appropriate :smile: